lib/lte_link_control/modules/ncellmeas.c

```diff
@@ -610,6 +610,17 @@ static int parse_ncellmeas(const char *at_response, struct lte_lc_cells_info *ce
 	return err;
 }
 
+static void dispatch_empty_neighbor_cell_meas_event(void)
```

Comment:
Would be good to write down what "empty neighbor cell meas event" means.
I guess it's just that the cell ID is `LTE_LC_CELL_EUTRAN_ID_INVALID`.
I would write this into commit and PR description as well as in the comment for `LTE_LC_EVT_NEIGHBOR_CELL_MEAS` and/or `lte_lc_neighbor_cell_measurement()`.

Also, `lte_lc_neighbor_cell_measurement()` comment could say that `LTE_LC_EVT_NEIGHBOR_CELL_MEAS` event is always sent, if `lte_lc_neighbor_cell_measurement()` has returned 0.

Reply:
I updated the PR, please have a look.